pandas/tests/test_resample.py

```diff
@@ -2192,6 +2192,17 @@ def test_resample_datetime_values(self):
         res = df['timestamp'].resample('2D').first()
         tm.assert_series_equal(res, exp)
 
+    def test_apply_to_empty_series(self):
+        # GH 14313
+        series = self.create_series()[:0]
+
+        for freq in ['M', 'D', 'H']:
+            result = series.resample(freq).apply(lambda x: 1)
+            expected = series.resample(freq).apply(np.sum)
+
+            assert result.name == expected.name
+            assert_series_equal(result, expected, check_dtype=False)
```

Comment:
Is there exist a way how to predict a `dtype` after [apply](https://github.com/pandas-dev/pandas/blob/master/pandas/core/resample.py#L403)?
In this example after aplying we have the next result:
```
In [1]: result
Out[1]: Series([], dtype: float64)

In [2]: result.index
Out[2]: Index([], dtype='object')
```

I decided to change the name and index type according to the issue. And we have a similar [test](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/test_resample.py#L761-L777) where we are not checking `dtype`